src/pazpar2.c

```diff
@@ -24,6 +24,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #include <winsock.h>
 #include <direct.h>
 #endif
+#if HAVE_STDLIB_H
```

Comment:
Checking `HAVE_STDLIB_H` is probably unnecessary; probably nobody cares about compilers that don't have it.

Reply:
You haven't checked `HAVE_STDLIB_H` anywhere else that you included <stdlib.h> so I've removed it in this PR as well.